src/components/VideoDetailsInline/VideoDetailsInline.tsx

```diff
@@ -20,7 +20,7 @@ const VideoDetailsInline: React.FC<Props> = ({ title, description, primaryMetada
   const breakpoint: Breakpoint = useBreakpoint();
   const isMobile = breakpoint === Breakpoint.xs;
 
-  const TitleComponent = typeof title === 'string' ? 'h2' : 'div';
+  const TitleComponent = typeof title === 'string' ? 'h1' : 'div';
```

Comment:
Is this indeed the title of the whole page?

Reply:
Yes, this is the title that's displayed underneath the Player:
![Screenshot 2023-10-19 at 15 44 59](https://github.com/Videodock/ott-web-app/assets/48496458/7ec24962-6b73-4fd7-be38-0f6ae25ac823)

Reply:
I see, makes sense!